frontend/src/components/Navbar/Navbar.test.tsx

```diff
@@ -0,0 +1,86 @@
+import '@testing-library/jest-dom'
+import {render, screen} from '@testing-library/react'
+import {expect, test, describe} from 'vitest'
+import { Navbar } from './Navbar'
+import { MemoryRouter } from "react-router";
+
+describe('Navbar', () => {
+  //DA DIFFERENZIARE TRA QUANDO L'UTENTE E' LOGGATO E QUANDO NON LO E'
+  test('Renders the navbar', () => {
+    render(
+      <MemoryRouter>
+        <Navbar />
+      </MemoryRouter>
+    );
+    expect(screen.getByRole('banner')).toBeInTheDocument()
+  });
+
+  test('Renders the home item', () => {
+    render(
+      <MemoryRouter>
+          <Navbar />
+      </MemoryRouter>
+  );
+      expect(screen.getByText('Home')).toBeInTheDocument()
+  });
+  
+  test('Renders the about us item', () => {
+    render(
+      <MemoryRouter>
+          <Navbar />
+      </MemoryRouter>
+    );
+      expect(screen.getByText('About Us')).toBeInTheDocument()
+  });
+
+  test('Renders the Agent Flow item', () => {
+    render(
+      <MemoryRouter>
+          <Navbar />
+      </MemoryRouter>
+    );
+      expect(screen.getByText('Agent Flow')).toBeInTheDocument()
+  });
+
+  test('Renders the logo', () => {
+    render(
+      <MemoryRouter>
+          <Navbar />
+      </MemoryRouter>
+    );
+      expect(screen.getByAltText('Logo Tech Minds')).toBeInTheDocument()
```

Comment:
Forse così è un pelo più preciso?

```suggestion
      expect(screen.getByRole("img")).toHaveAttribute(
      "alt",
      "Logo Tech Minds"
    );
```

Reply:
sad, ci penso io